src/realm/index_string.cpp

```diff
@@ -589,6 +589,10 @@ IndexArray* StringIndex::create_node(Allocator& alloc, bool is_leaf)
     return top.release();
 }
 
+ref_type StringIndex::create_empty(Allocator& alloc)
```

Comment:
table.cpp:1411. Here we create empty columns when getting the subtable out of degenerate state.